ex02.ex

```diff
@@ -3,62 +3,35 @@ Code.load_file "support.ex"
 
 defmodule Ex02 do
   use ExUnit.Case
-  import Support
 
-  ##############################################################################
-  # 2: 3 questions,  15 points available
-  ##############################################################################
-
-  # These exercises manipulate lists using anonymous functions.
-  # Replace the text
-  #
-  #      your_anonymous_function(...)
-  #
-  # with your solution. Note that even though the placeholder
-  # takes just one line, your solution may take several.
-  #
-  # You can test your answers by running:
-  #
-  #     elixir ex02.ex
-
-
-  ##################
-  # 2.1:  5 points #
   ########################################################################
   # Write an anonymous function that takes two numbers and returns a two #
   # element list. The first element of the list will be the sum of the   #
   # numbers, and second should be the difference                         #
   ########################################################################
 
-  list2a = your_anonymous_function(1, 2)
-
+  list2a = fn (x, y) -> [x + y, x - y] end
```

Comment:
I like the spacing you've used here.

If there were also multiplictive operators in the expressions, you could leave out the spacing around them, so I'd personally write:

~~~
a = 1 + b
~~~

and

~~~
a = 1 + 2*b
~~~